frontend/src/components/TopicCard/index.js

```diff
@@ -53,7 +52,7 @@ class ReactTopicCard extends Component {
                 authorizedToEdit={authorizedToEdit}
                 updateTopic={wrappedUpdateTopic}
               />
-            {Util.isTester(currentUser) && <Follow isFollowing={isFollowing} onTopicFollow={wrappedTopicFollow} />}
+            <Follow isFollowing={isFollowing} onTopicFollow={wrappedTopicFollow} />
```

Comment:
is the styling on this still a bit ugly? can we upgrade a bit as part of this? even just some minimal changes

Reply:
It's not ugly, but it's also not pretty. I'll see if Pegah can just help me improve how it looks and works a little bit